### PR TITLE
Add logout functionality

### PR DESCRIPTION
**Pull request by Alex7Li**
Adds the logout/login button. and also a icon
Capture

### CODE REVIEW

**Comment by Alex7Li** [comment]
I think the build is failing because of https://github.com/samuelstevens/watsong/issues/39. Anyone else having this issue?

File: watsong/spotify.py [hunk 1]
```diff
@@ -299,6 +290,21 @@ def create_playlist(
     )
 
 
+def logout() -> str:
+    """
+    Change who you are logged in as.
+    Return "" on successful change, else return a string for the error
+    """
+    cache_file_path = f"{os.getcwd()}\\.cache"
```
Comment (samuelstevens) [inline watsong/spotify.py:298]: Use this for platform-independence: https://docs.python.org/3/library/os.path.html#os.path.join

**Review by samuelstevens — APPROVED** [review]

File: watsong/spotify.py (hunk 1 above)
Comment (Alex7Li) [reply watsong/spotify.py]: Thanks, good catch!

**Review by samuelstevens — APPROVED** [review]
Yeet this into master

**Comment by samuelstevens** [comment]
Nice nice

**Pull request merged**